rachellegelden/Home-Together-Canada
# HTC-183: React routing

# [Issue 183](https://github.com/rachellegelden/Home-Together-Canada/issues/183)

closes #183 

## Summary
Added links and routing to the website using `react-router-dom`on the following files:
- `Home.js`
- `Header.js`
- `Footer.js`
- `App.js`

## Relevant Motivation & Context
The website is a single page set up. Using `react-router-dom`, we can navigate the user to different "pages" by swapping the components. 

## Testing Instructions
run `npm install` on the client folder to install `react-router-dom`. Load up the webiste using `npm start`. 
1. Using the Header, navigate to the different areas of the website. 
2. Using the Footer, navigate to the different areas of the website. 
3. Using manual URL entry, navigate to the different areas of the website. (look at the code to see different ULRs i.e about/faq etc.

## Developer checklist prior to opening this pull request:


## Reviewer
